app/src/main/kotlin/compiler/ast/AstNode.kt

```diff
@@ -28,7 +28,7 @@ sealed interface AstNode {
                     is Expression.ArrayAllocation -> {
                         var arrayValues = this.initialization.joinToString { it.toSimpleString() }
                         if (this.initializationType == Expression.ArrayAllocation.InitializationType.ONE_VALUE) arrayValues += "..."
-                        "ciąg ${this.type}[${this.size.toSimpleString()}]{ $arrayValues }"
+                        "alokacja ${this.elementType}[${this.size.toSimpleString()}]{ $arrayValues }"
```

Comment:
I think we (finally) agreed on "ciąg" as the allocation keyword, so update this here and in code snippets in tests.